src/Components/Shared/LandingPage.tsx

```diff
@@ -13,8 +14,12 @@ export default function LandingPage() {
 // We're using the Implicit Grant flow
 function logInButton(): ReactNode {
     return (
-        <a href={authEndpoint + "?client_id=" + clientId + "&response_type=token&redirect_uri=" + redirectUri}>
-            <button>Spotify Login</button>
-        </a>
+        <AuthenticationContext.Consumer>
```

Comment:
We need to have some sort of indication to the user if we can't log in, see my PR for a suggestion